src/components/RichTextEditor/serializer/nodes/link.ts

```diff
@@ -13,7 +13,9 @@ export const linkNode = (node: TElement, children: string, designTokens: DesignT
             chosenLink?.openInNewTab ? '_blank' : '_self'
         } href="${escapeHtml(chosenLink?.searchResult?.link)}">${children}</a>`;
     }
-    return `<a class="${defaultClassNames}" style="${reactCssPropsToCss(designTokens.link)}" target="${
-        node?.target ?? '_self'
-    }" href="${escapeHtml(node.url as string)}">${children}</a>`;
+    return `<a class="${defaultClassNames}" style="${reactCssPropsToCss(
+        designTokens.link,
+    )}" onClick="((event)=>{event.stopPropagation()})(event)" target="${node?.target ?? '_blank'}" href="${escapeHtml(
```

Comment:
The wrapping container needs to have click action just when you click on the link inside of this container should not trigger the wrapper container event.

Reply:
Updated it.

Reply:
I just changed the target, because the plate has undefined for _blank :) the is the bug in plate, which i solved and will be in in the future when we update the plate to the last version.